.github/workflows/slack.yaml

```diff
@@ -0,0 +1,36 @@
+# Send a message to development when the docker image changes!
+# Firstly made by Finn Stevenson!
+
+name: Docker Slack Message
+
+# Control when this will run!
+on:
+  #testing on pull request, sends a message everytime
+  # pull_request:
+  #   branches: [main]
+  #   paths:
+  #     - 'docker/**'
+
+  push:
+    branches:
+      - main
+    paths:
+      - 'docker/**'
```

Comment:
We shouldn't run this workflow on the files relating to the robocup docker image, these are `Dockerfile.nugus_sim` and `nugus_sim/`. I believe you can exclude them by prepending a `!`.

Reply:
I can tell it to ignore certain directories. I shall implement this change!

Reply:
Applied path rules to ignore the robocup files.